docs-src/concepts/what-are-actvitiy-events.md

```diff
@@ -0,0 +1,28 @@
+---
+id: what-are-activity-events
+title: What are Activity Events?
+sidebar_label: Activity Events
+description: There are seven Activity Events that are added to History at different points in an Activity Execution.
+tags:
+  - term
+  - explanation
+---
+
+There are seven Activity-related [Events](/workflows#event) that are added to History at different points in an Activity Execution.
+
+There are two important things to note when matching Activity Execution lifecycle to Activity Events:
+
+- In Event names, "ActivityTask" refers to an [Activity Execution](/activities#activity-execution), not an [Activity Task](/tasks#activity-task).
```

Comment:
I'm not sure this is a useful or accurate thing to say. They are not identical, but the execution is the successful dispatch of the task. Mostly I think it is actually talking about the task. E.g. if an activity task is never delivered, times out, and an ActivityTaskTimedOut gets written, it is the task that has timed out, not the execution (no execution ever happened)

Reply:
By "Activity Execution", I'm talking about this, which doesn't necessarily include any Activity code execution: https://docs.temporal.io/activities#activity-execution

Reply:
And by "Activty Task" I'm talking about the docs definition: https://docs.temporal.io/tasks#activity-task

Wondering if your comment still holds

Reply:
Yeah, I read the links before I made my comment

I think it's more coherent (and I believe matches the original intention, though I could be wrong) to think about the events as talking about tasks, but then note that some logical events are omitted from history as an optimization, as compared to thinking of the events as talking about the full execution